site/ic_data_repo/import_imperial_users.py

```diff
@@ -103,6 +105,16 @@
 _QueryParameters = UsersRequestBuilder.UsersRequestBuilderGetQueryParameters
 
 
+def _get_default_logger() -> Logger:
+    """Get a default logger for this module which just prints to stdout."""
+    logger = logging.getLogger(__name__)
+    logger.setLevel(logging.INFO)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.INFO)
+    logger.addHandler(ch)
+    return logger
```

Comment:
The below is pretty equivalent for our purposes.

```suggestion
    logging.basicConfig(level=logging.INFO)
    logger = logging.getLogger(__name__)
    return logger
```